docs/src/Fundamentals/README.md

```diff
@@ -6,12 +6,12 @@
 
 Frequency provides a shared global state to make interoperability and user control fundamental to the internet.
 Applications provide unique experiences to their users while accessing the content and graph connections from other applications.
-Just like how the internet is a shared global network, your applications can break out of their artificial boundaries.
+In the same way the internet is a shared global network, Frequency allows your applications to move past artificial boundaries.
```

Comment:
nit: prioritize Frequency in the messaging. Also, if we're making comparisons, perhaps the network stack itself is more apt than just "the internet"?
```suggestion
Frequency enables your applications to interact with this shared global state seamlessly, in the same way that modern networking software allowed isolated computers to interact seamlessly over a global network.
```